connector/gitlab/gitlab_test.go

```diff
@@ -0,0 +1,29 @@
+package gitlab
+
+import (
+	"testing"
+)
+
+func TestNextURL(t *testing.T) {
+	linkHeader := "<https://gitlab.com/api/v4/groups?page=3&per_page=20>; rel=\"next\", " +
```

Comment:
nit: we generally prefer table driven tests https://github.com/golang/go/wiki/TableDrivenTests

If you want to add that here that works, if not I'm fine with the test as is.

Reply:
I'll fix! Thanks!